cumulus/polkadot-omni-node/lib/src/common/spec.rs

```diff
@@ -100,15 +131,28 @@ pub(crate) trait BaseNodeSpec {
 		ParachainClient<Self::Block, Self::RuntimeApi>,
 	>;
 
-	type BuildImportQueue: BuildImportQueue<Self::Block, Self::RuntimeApi>;
+	type BuildImportQueue: BuildImportQueue<
+		Self::Block,
+		Self::RuntimeApi,
+		<Self::WrapBlockImport as InitBlockImport<Self::Block, Self::RuntimeApi>>::BlockImport,
+	>;
+
+	type WrapBlockImport: InitBlockImport<Self::Block, Self::RuntimeApi>;
```

Comment:
nit: naming again: maybe `WrappedBlockImport`?

Reply:
It was first called `WrapBlockImport`, but then I decided for a different name. Apparently I failed to rename it everywhere :P Init in the context is more correct imo.